Sources/PackageModel/Target.swift

```diff
@@ -104,16 +108,34 @@ public class Target: PolymorphicCodableProtocol {
     ///
     /// NOTE: This name is not the language-level target (i.e., the importable
     /// name) name in many cases, instead use c99name if you need uniqueness.
-    public let name: String
+    public private(set) var name: String
+
+    /// Module aliases needed to build this target.
+    public private(set) var moduleAliases: [String: String]?
 
+    /// Set module aliases needed to build this target.
+    /// shouldRename True if the target itself should be renamed
+    public func setModuleAliases(name: String, alias: String, shouldRename: Bool = false) {
```

Comment:
should this be a setter instead of a func?

Reply:
can you explain a bit when `shouldRename` is used, it's interesting that its the caller responsibility to decide that?

Reply:
It's set to true if the target name is the one that needs to be aliased as opposed to its dependency name; will look into autodetecting it / removing the param.